app/controllers/pages_controller.rb

```diff
@@ -3,7 +3,7 @@ def home
   end
 
   def about
-    @members = Member.all
+    @members = Member.all.sort_by(&:member_role)
```

Comment:
@SamLau95 Could you explain why interacting with the DB directly messes the order up? I thought that even if you do interact with the DB directly, with Member.all we're able to get all the Members, and then sort_by simply organizes them in ascending order?